### ISSUE TITLE
Enhancement request: Preference to save valid queries only

### ISSUE THREAD

**Issue by teneruffian** [jakob/Postico#214]
Postico is a great product and I use it throughout my working day.  However, I end up with a lot of invalid SQL in the history (usually through typos).  Would it be possible to have a new preference: "Save valid SQL only" ?
Thanks.

**Comment by jakob** [comment]
Good idea. I think I've heard this suggestion before. I'll see if I can find a way to do this...
